ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/backups-details.tsx

```diff
@@ -172,6 +195,12 @@ export const BackupsDetails = ({
           dataTestId="pitr"
           title={Messages.titles.pitr}
           loading={loading}
+          actionButtonProps={{
+            onClick: () => {
+              setOpenEditModal(true);
+            },
+          }}
+          editable={canUpdateDb && !restoring && !deleting}
```

Comment:
if the number of statuses to check gets bigger, we'll end up with lots of different booleans.
Something like:
```
//...
const editable = ![DbClusterStatus.restoring, DbClusterStatus.deleting].includes(dbCluster?.status?.status);
//...
editable={editable}
```
Might be more readable.

Reply:
Same for the others similar below

Reply:
Agreed, but I would keep restoring and deleting as separate consts for better visibility, what do you think?

Reply:
```
  const restoringOrDeleting = ![
    DbClusterStatus.restoring,
    DbClusterStatus.deleting,
  ].includes(dbCluster?.status?.status!);

  const editable = canUpdateDb && !restoringOrDeleting && canUpdateMonitoring;
```

Reply:
sure, let's go with that